cmp/relativetimestamp/RelativeTimestamp.js

```diff
@@ -57,25 +59,35 @@ export class RelativeTimestamp extends Component {
 
     baseClassName = 'xh-relative-timestamp';
 
-    @observable relativeTimeString;
+    @observable stampFormat = {};
     timer = null;
-
     render() {
+        const {textItem, prefix, prefixMargin} = this.stampFormat;
         return box({
             ...this.getLayoutProps(),
             className: this.getClassName(),
-            item: this.relativeTimeString
+            items: [
+                span({
+                    item: prefix,
+                    style: {marginRight: prefixMargin}
+                }),
+                span({
+                    className: 'xh-title-tip',
+                    item: textItem,
+                    title: fmtDateTime(this.props.timestamp),
+                })
```

Comment:
See comment below which should simplify / make this fall out anyway, but it would be natural / friendly to have the tooltip appear if the user happened to hover over the prefix as well.

Reply:
I originally embedded prefix within the string as you suggest, but then probably started to overthink it. If you did want to offset the prefix, you could just leave the prefix option null and create a separate <span> label. Will make these changes, thanks.